docs/README.md

```diff
@@ -157,12 +159,23 @@ Check out [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-dat
 
 ### Configuring Lookback
 
-The lookback strategy determines how the agent handles existing files on startup. This strategy is determined by `LOGDNA_LOOKBACK`. The set of valid values for this option are:
+The lookback strategy determines how the agent handles existing files on startup. This strategy is determined by `LOGDNA_LOOKBACK`.
+
+The valid values for this option are:
+   * When set to **`none`**: lookback is disabled, and LogDNA continues to read from where the file ended at the time of the stop/restart action. 
```

Comment:
In technical terms `none` means the Agent will go to the end of the file, ignoring previously written lines, and it will read new lines of the file as those are added (tailing).

```suggestion
   * When set to **`none`**: lookback is disabled, and LogDNA Agent will reading new lines as those are added to the file, ignoring the lines that were wrote before the time the Agent restarted.
```

Reply:
Ah, thanks much ore clear. Two grammar changes though please:

"... Agent will start reading new..." 
"...lines that were written before..."